scripts/reset_iks_api_key.sh

```diff
@@ -51,8 +52,13 @@ fetch_data
 
 if [ "${reset}" == true ]; then
     if [ "$PRIVATE_ENV" = true ]; then
-        RESET_URL="https://private.$REGION.containers.cloud.ibm.com/v1/keys"
-        curl -H "accept: application/json" -H "Authorization: $IAM_TOKEN" -H "X-Auth-Resource-Group: $RESOURCE_GROUP_ID" -X POST "$RESET_URL"
+        if [ "$CLUSTER_ENDPOINT" == "private" ] || [ "$CLUSTER_ENDPOINT" == "default" ]; then
```

Comment:
hmm, at one point (I think 4.13) `vpe` became the default - so that would break this logic right?

Reply:
<img width="693" alt="Screenshot 2024-07-12 at 4 09 41 PM" src="https://github.com/user-attachments/assets/8a1eddd8-c5d1-4780-9f95-2ae197f600fa">
I ran some tests on ocp-version 4.15, 
When we set `default`, it still uses the public endpoint by default, unless we disable the public endpoint endpoint. In that case `default` value will use the private endpoint.
And only when we set value as `vpe`, VPE endpoint was used.

Reply:
yea thats because in 4.14, they changed the default back to public / private. but in 4.13 default was VPE. I don't know if they plan to go back to this, but since we still support 4.13 in this module, you might want to verify your logic

Reply:
![Screenshot 2024-07-15 at 11 53 54 AM](https://github.com/user-attachments/assets/d088b24e-69e2-43bb-92a4-67449b2bf59f)
I tested using ocp version 4.13 as well. It is using public link for both `null` and `default` value

Reply:
@Aashiq-J but when you do a private only cluster on 4.13, it will use VPE endpoint as far as I remember